squad/frontend/templates/squad/_builds_table.html

```diff
@@ -17,8 +17,11 @@
         </th>
     </tr>
     {% for build in builds %}
-    <tr>
-        <td><a href="{% project_url build %}">{{build.version}}</a></td>
+    <tr class="{% if build.status.tests_fail > 0 %}fail{% else %}pass{% endif %}">
```

Comment:
I don't think I agree with this assumption. Build failure is different than test failure. I don't like the idea of test failures to be displayed as the text colour. For example all LKFT builds would be marked 'fail' with this code.

Reply:
my assumption is that SQUAD cares about *tests*, and not builds, and my goal is to highlight which builds have failed tests to make it easier for people going over the data to find things that need to be fixed.

Reply:
This will make all LKFT red as there will most likely always be some tests that are failed. So please change that unless we can make it configurable for project.

Reply:
I have played with this and explored hiding the zeros instead. please let me know what you think:

![build-list](https://user-images.githubusercontent.com/30027/30557514-f4bfd2aa-9c84-11e7-8363-0d81cafe149f.gif)